### DISCUSSION TITLE
Home cant be blocked corret

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#3442] [Q&A] [answered]
**Question  for help or a Request**
I made this issues for an other plugin called Redprotect. 
Its a protection plugin that allows managing the protections with flags that can be setup like in Worldguard just with a nice gui like in Ressidence.
One of the flags should block setting a homepoint. -> Well this works so far but only if the user wanne use the regular command /sethome.
If he works arround with using /cmi sethome then this wont be blocked.
We asked the dev of Redptrotect for a solution. -> If i understood him correct he said that he just "scans" for the regular Essential commands what is /sethome. 

Could you help us out? Maybe Zrips can write a post there to explain why this is not the best solution or maybe write him what he can do to prevent it in a better way so also /cmi sethome would be blocked. 

Or maybe Zrips can "add" to CMI a solution (like wehn /sethome is blocked by a protection plugin then /cmi sethome also cant be useeld? - Maybe something like a optional hook?
Im sure this Problem also may affect other protection plugins that just block /sethome with a flag... 

Here is the link from the Redprotect github issue:
https://github.com/FabioZumbi12/RedProtect/issues/590

**Comment by Zrips** [comment, marked as answer]
Maybe some what better solution would be ti fire block place event at that location to make sure that player can build there and if he cant then we can block home creation. This could resolve issue, but could introduce some other posible issues where players could want to set home location but is not allowed due to some protection, like near pvp arena.
